src/components/TopNav/index.js

```diff
@@ -0,0 +1,7 @@
+//will display the ChevronLeft icon on the left if "backIcon" is true
```

Comment:
this page should explain how to use the props to make the TopNav be reusable but let me know if something doesn't make sense or it doesn't work for you